src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/AsylumCaseDefinition.java

```diff
@@ -128,7 +128,10 @@
         "submitHearingRequirementsAvailable", new TypeReference<YesOrNo>(){}),
 
     SUBMISSION_OUT_OF_TIME(
-        "submissionOutOfTime", new TypeReference<YesOrNo>(){})
+        "submissionOutOfTime", new TypeReference<YesOrNo>(){}),
+
+    CCD_CASE_ID(
```

Comment:
do you have new field in definitions? there is case id in `CaseDetails` object. If it is the same id, there is no need to create another field

Reply:
This field should have been removed after a change to the template content in Confluence (fyi, it was originally used to build up a URL string to the Documents tab in the online service). I've removed, well spotted!